### PR TITLE
refactor: Uses types from supertokens-website for session recipe

### PR DESCRIPTION
**Pull request by nkshah2**
## Summary
- Uses input type from supertokens-website instead of redeclaring it

## Related PRs (These should be merged before this one)
- https://github.com/supertokens/supertokens-web-js/pull/23

## Remaining TODOs
- [x] Verify size-limit

### CODE REVIEW

File: lib/ts/recipe/thirdpartypasswordless/types.ts [hunk 1]
```diff
@@ -44,7 +49,7 @@ export type UserInput = {
             builder: OverrideableBuilder<RecipeInterface>
         ) => RecipeInterface;
     };
-};
+} & RecipeModuleUserInput<PreAndPostAPIHookAction>;
```
Comment (rishabhpoddar) [inline lib/ts/recipe/thirdpartypasswordless/types.ts:52]: How is this change related to this PR?

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: lib/ts/recipe/thirdpartypasswordless/types.ts (hunk 1 above)
Comment (nkshah2) [reply lib/ts/recipe/thirdpartypasswordless/types.ts:52]: Its not, theres another PR (ive linked it in the PR description) that should be merged before this one
